setup.cfg

```diff
@@ -1,3 +1,36 @@
+[metadata]
+name = lsst_sphgeom
+description = A Spherical Geometry Library
+author = LSST Data Management
+author_email = support@lsst.org
+license = GPLv3
+url = https://github.com/lsst/sphgeom
+classifiers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Topic :: Scientific/Engineering :: Astronomy
+
+[options]
+zip_safe = False
+package_dir=
+    =python
+packages=find:
+setup_requires =
+  pybind11 >=2.5.0
+  setuptools_cpp >=0.0.1
+install_requires =
+  pyyaml >=5.1
```

Comment:
Is this true?  I'd have thought that the functionality that requires yaml does no harm and cannot even be invoked unless pyyaml is installed.

Reply:
Ah, just looked at the imports - I think the above is only true if we put some import guards either in `__init__.py` or `_yaml.py`, which _might_ be worth doing.  Depends, I guess, on whether declaring it here makes sure pyyaml appears first in some dependency-ordering somewhere when both are present.

Reply:
I could lower the dependency version a bit but v5 added a bunch more loaders and I've have to do lots of import protection on them. Since we've had v5 for a while now there didn't seem to be any harm in requiring v5.

Reply:
I was actually just suggesting that this package didn't need to depend on pyyaml at all, from the perspective that there is no functionality it provides that cannot be used without pyyaml, except for functionality that can only be invoked _through_ pyyaml.  But I'm not sure that's worth guarding the imports.

Reply:
The tests will fail if the Yaml package is not there and the imports inside the module are not protected. Deliberately not installing it seems to be more work than requiring it

Reply:
On second thoughts you probably should protect them. You are right that it’s only usable if the module is there and if it’s not there you never care.